### PR TITLE
Add MotaJS

### PR DESCRIPTION
**Pull request by motapc97**

### CODE REVIEW

File: files/motajs/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "motajs",
+  "repo": "motapc97/motajs",
+  "files": {
+    "basePath": "main/",
+    "exclude": [ "mota.docs.json" ]
```
Comment (jimaek) [inline files/motajs/update.json]: Can you add mota.js and mota.css as well? We only need minified files

**Comment by motapc97** [comment]
Sorry about that. I updated now "update.json" as you asked.

**Comment by jimaek** [comment]
Great, can you squash your commits into 1 now? Then its good to go

**Comment by motapc97** [comment]
I used Github website to add and edit the files because copying the entire repository to my computer is too difficult...

Is there a way to squash the commits without cloning the repository?

(sorry, for my english)

**Comment by motapc97** [comment]
Done!

**Pull request merged**